packages/shorebird_cli/test/src/commands/patch_new/patch_new_command_test.dart

```diff
@@ -419,6 +427,19 @@ void main() {
             () => patcher.buildPatchArtifact(),
           ]);
         });
+
+        test('updates cache with both default and release Flutter revisions',
```

Comment:
Do we need to do the same thing for release new?

Reply:
Done.